core/src/components/modal/modal.tsx

```diff
@@ -560,6 +582,43 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return eventMethod(this.el, 'ionModalWillDismiss');
   }
 
+  /**
+   * Move a sheet style modal to a specific breakpoint.
+   */
+  @Method()
+  async setBreakpoint(breakpoint: number): Promise<void> {
+    if (!this.isSheetModal || !this.breakpoints!.includes(breakpoint) || this.currentBreakpoint === breakpoint) {
+      return
```

Comment:
~ Can we add a console warning here to inform developers when they are trying to set a breakpoint value on the unsupported cases?

```ts
if (!this.isSheetModal) {
  console.warning('setCurrentBreakpoint is only supported on sheet modals.');
  return;
}
if (!this.breakpoints!.includes(breakpoint)) {
  console.warn(`Attempted to set invalid breakpoint value ${breakpoint}. Please double check that the breakpoint value is part of your defined breakpoints.`);
  return;
}
if (this.currentBreakpoint === breakpoint) {
  return;
}
``